schema/materials_category/defects/configuration.json

```diff
@@ -0,0 +1,16 @@
+{
+    "$id": "materials-category/defects/configuration",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Base Defect Configuration Schema",
+    "description": "Base configuration for all defect types",
+    "type": "object",
+    "required": [
+        "crystal"
+    ],
+    "properties": {
+        "crystal": {
+            "description": "The base crystal material for the defect",
+            "$ref": "../../material.json"
+        }
```

Comment:
We should add `dimensionality` and organize the folders:

```
defects/by_dimensionality/zero_dimensional/
defects/by_dimensionality/one_dimensional/
defects/by_dimensionality/two_dimensional/
defects/by_host_type/monolayer/
defects/by_host_type/slab/
defects/by_host_type/bulk/
```

The importance is increasing toward the bottom